test/functional/autocmd/helpfile_spec.lua

```diff
@@ -0,0 +1,22 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local eq = helpers.eq
+local eval = helpers.eval
+local clear = helpers.clear
+local meths = helpers.meths
+local expect = helpers.expect
+local command = helpers.command
+local exc_exec = helpers.exc_exec
+local curbufmeths = helpers.curbufmeths
+
+describe('autocmd FileType', function()
+    before_each(clear)
+
+    it("triggered by nvim_command('help <cmd>')", function()
```

Comment:
there is no need to mention the API function, is is incidential, instead something like `'is triggered by :help only once'`

Reply:
ok done